### DISCUSSION TITLE
Docker Acquisition

### DISCUSSION THREAD

**Discussion by austince** [testcontainers/testcontainers-go#1991] [General]
With the recent [acquisition of AtomicJar by Docker](https://techcrunch.com/2023/12/11/docker-acquires-atomicjar-a-testing-startup-that-raised-25m-in-january/), are there any bits of news from the team how that shapes the roadmap of this project?

**Comment by kiview** [comment]
There is no change on any of our current OSS plans or roadmaps. Our OSS projects are only impacted insofar as this deal will further accelerate our OSS work through more resources and better synergies with related projects.

**Reply by austince** [reply]
Thanks! Probably too early to make this call, but wondering what the stance will be towards support for non-docker container runtimes, e.g. podman?

**Reply by mdelapenya** [reply]
Hey @austince there is some explicit code regarding Podman, like a `PodmanProvider`, which should have been aligned with the rest of the Testcontainers libraries, that work with Podman without any explicit support. Therefore, the current auto-discovery mechanism should detect Podman running in Docker compatibility mode just as a regular Docker provider. And if this is not directly possible, the flexible configuration mechanisms (such as configuration of DOCKER_HOST or TESTCONTAINERS_DOCKER_SOCKET_OVERRIDE) should be used to to support other runtimes, for which their API is available at different locations.

That is the same approach we use for integrating against other container runtimes in our Testcontainers language implementations.
